src/rustdoc_gen.rs

```diff
@@ -658,10 +658,14 @@ impl RustdocGenerator for RustdocFromRegistry {
         rustdoc_cmd: &RustdocCommand,
         crate_data: CrateDataForRustdoc,
     ) -> anyhow::Result<PathBuf> {
-        let crate_ = self
-            .index
-            .crate_(crate_data.name)
-            .with_context(|| anyhow::format_err!("{} not found in registry", crate_data.name))?;
+        let crate_ = self.index.crate_(crate_data.name).with_context(|| {
+            anyhow::format_err!(
+                "{} not found in registry (crates.io). \
+       For workarounds check \
```

Comment:
```suggestion
        For workarounds check \
```
just an alignment fix